readme.md

```diff
@@ -47,38 +47,89 @@ To understand the philosophy of self-sovereign identity these starting guides an
 
 ## 📄 Technical Specifications
 
+Here you will find an overview of the technical specifications, grouped by level and sorted by community.
+
+An overview of the technical specifications can be found in [this simplified map](https://github.com/decentralized-identity/decentralized-identity.github.io/blob/master/assets/map-of-adjacent-orgs-and-specs--sept-2020--one-sided.pdf) and [here](https://github.com/decentralized-identity/decentralized-identity.github.io/blob/master/assets/crosscommunity-architecture-survey-oct-2020.pdf) in a more detailed fashion.  
+
+### Verifiable Credentials / Trusted Data Storage & Exchange
```

Comment:
Suggested order: first DIDs, then VCs then DIDComm. To order them relating to their dependence on each other (VCs often use DIDs, DIDComm often uses VCs and DIDS).